salt/states/pkgrepo.py

```diff
@@ -106,6 +106,18 @@ def __virtual__():
     return 'pkg.mod_repo' in __salt__
 
 
+def _strip_uri(repo):
```

Comment:
Rather than adding this here and having it in `aptpky.py`, why don't we made a util function instead?